README.md

```diff
@@ -17,17 +17,17 @@ guaranteed nor implied!
 ### From NPM Package
 
 ```
-npm install hellosign-sdk@github:HelloFax/hellosign-nodejs-sdk#openapi
+npm install hellosign-sdk@6.0.0-beta
 ```
 
 ### From Repo
 
 * Clone this repo locally
 * Run `npm pack`
-* The generated file will be named similar to `hellosign-sdk-6.0.0.tgz`
+* The generated file will be named similar to `hellosign-sdk-6.0.0-beta.tgz`
 * Move this file to your project directory
 * Add the following to your project's `package.json` under `dependencies`:
-* `"hellosign-sdk": "file:hellosign-sdk-6.0.0.tgz"`
```

Comment:
This line is for importing an archive file from local filesystem, I think it requires the `file:` part.